elastic/kibana
# [Exploratory view] filter by summary.up : * for availability metric

## Summary

Resolves https://github.com/elastic/kibana/issues/143672

Adds global summary filter to availability metric

Also fixes an error where an invalid `orderBy` attribute was being applied to formula columns, causing invalid aggregation errors.

<img width="1374" alt="Screen Shot 2022-11-07 at 10 36 07 PM" src="https://user-images.githubusercontent.com/11356435/200470672-b1d26892-fb0b-4978-a76a-fee01fc8570b.png">

### Testing
1. Create an always down browser monitor
2. Navigate to exploratory view
3. Filter for that monitor
4. Ensure that availability is marked as 0%
